packages/lib/src/vis/heatmap/HeatmapMesh.tsx

```diff
@@ -143,7 +145,7 @@ function HeatmapMesh(props: Props) {
   };
 
   return (
-    <VisMesh scale={[1, ordinateConfig.flip ? -1 : 1, 1]} size={size}>
+    <VisMesh scale={scale ?? [1, ordinateConfig.flip ? -1 : 1, 1]} size={size}>
```

Comment:
I did it this way to minimize the changes to `HeatmapMesh`, but I have the feeling that the axes flip would be best handle in the vis (no matter how), so that the HeatmapMesh` is as close as possible to a basic three.js component (i.e., not relying on the axes context).

Reply:
Agreed. Given your changes, I think it is best to pass the `[1, ordinateConfig.flip ? -1 : 1, 1]` in `HeatmapVis` through the  `scale` prop.